lib/customer.rb

```diff
@@ -0,0 +1,68 @@
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+  @@new_customers = [] #will initialize when this file is called
+
```

Comment:
Regarding the comment on line 6, I don't know that this is true. I don't see this variable written to anywhere, which means that in `save` you'll write an empty list of customers to disk.